DEVELOPMENT.md

```diff
@@ -47,6 +47,24 @@ Spec changes should be approved by a majority of the committers.  Approval can b
 
 No change should be approved until there is documentation for it, supplied in an accompanying PR.
 
+## Draft Features
+
+Where suitable, features will be introduced as draft but OAI approved extensions.
+By introducing new features this way we enable new features to be designed, documented and then implemented by tools that are interested in the feature, without putting the burden of implementation on all tooling.
+If the feature is successfully implemented and there is demonstrable value added by the feature, it will become a candidate for inclusion in a future release of the specification, at which point all tools will be expected to support the feature.
+
+Draft feature extensions are identified by the `x-oas-draft-` prefix and can only be used where existing extensions are permitted.
+This ensures no exising tooling will affected by the introduction of the draft feature.
```

Comment:
"existing"

Reply:
Is it worth calling out that draft extensions can only be promoted into minor or major (breaking) versions of the spec, not patches? That may be deemed obvious from the rest of the document though.